include/gatekeeper_gk.h

```diff
@@ -65,6 +65,16 @@ struct gk_measurement_metrics {
 	uint64_t tot_pkts_num_dropped;
 	/* Total size in bytes of packets dropped. */
 	uint64_t tot_pkts_size_dropped;
+	/* Total number of packets distributed to other blocks. */
+	uint64_t tot_pkts_num_distributed;
```

Comment:
Better comment since this condition counts ARP and ACLs (see another comment):
```
/*
 * Total number of packets distributed to other blocks.
 * It includes ARP packets handled to the LLS block, and packets handed off to the IPv4 and IPv6 ACLs.
 */
```
Break the comment into lines as needed.